src/renderer/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useMemo, useState } from 'react'
+
+import { Tabs as ATabs } from 'antd'
+
+import * as Styled from './Tabs.styles'
+
+type Props = {
+  defaultTabIndex?: number
+  tabs: { label: string; key: string; content: React.ReactNode }[]
```

Comment:
i'd like to leave it 'private'. i believe there is no need to extract this type for outside components. in case any component wants use `Tabs` component TS compiler will check props' typings anyway

Reply:
Ok.